apps/prairielearn/src/sync/course-db.ts

```diff
@@ -506,10 +510,13 @@ export async function loadCourseInfo({
   tagsInUse: Set<string>;
   sharingEnabled: boolean;
 }): Promise<InfoFile<CourseJson>> {
-  const maybeNullLoadedData: InfoFile<CourseJson> | null = await loadInfoFile({
+  const maybeNullLoadedData = await loadAndValidateJson({
```

Comment:
I use this function as the `loadAndValidateJson` version does the zod re-parse as well.